hooks/useFilters.ts

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react'
+import { useQuery } from 'react-query'
+import { useDeepCompareMemo } from 'use-deep-compare'
+import { getAllTaxonomies } from 'api'
+import { TaxonomyTerm } from 'models'
+
+export interface FiltersHandler<T> {
```

Comment:
SO THIS IS WHERE MOST OF THE FILTERING LOGIC NOW LIVES?

Reply:
(ignore caps lock)

Reply:
Lol, yep!